src/Nethermind/Nethermind.Config/NoCategoryConfig.cs

```diff
@@ -12,7 +12,7 @@ public class NoCategoryConfig : INoCategoryConfig
         public string DataDir { get; set; }
         public string ConfigsDirectory { get; set; }
         public string BaseDbPath { get; set; }
-        public string LogLevel { get; set; }
```

Comment:
Why change that?

Reply:
Because it was wrong here. The command line option is actually "Log".